packages/forklift-console-plugin/src/components/FormErrorHelperText.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { FC } from 'react';
+import { FieldError } from 'react-hook-form';
+
+import { FormHelperText, HelperText, HelperTextItem } from '@patternfly/react-core';
+import { ExclamationCircleIcon } from '@patternfly/react-icons';
+
+type FormErrorHelperTextProps = {
+  error: Partial<FieldError>;
+};
+
+export const FormErrorHelperText: FC<FormErrorHelperTextProps> = ({ error }) => {
+  return error ? (
+    <FormHelperText>
+      <HelperText>
+        <HelperTextItem icon={<ExclamationCircleIcon />} variant="error">
+          {error?.message?.toString()}
+        </HelperTextItem>
+      </HelperText>
+    </FormHelperText>
+  ) : null;
```

Comment:
```suggestion
if (isEmpty(error)) return null

  return (
    <FormHelperText>
      <HelperText>
        <HelperTextItem icon={<ExclamationCircleIcon />} variant="error">
          {error?.message?.toString()}
        </HelperTextItem>
      </HelperText>
    </FormHelperText>
  )
```

Reply:
What benefit is there to using the "isEmpty" helper here?

Reply:
Also, I don't agree that non-bracketed if-statements are more readable.